docs/rest-api/concepts/scan-guidance.md

```diff
@@ -72,15 +80,26 @@ After your application receives a notification through a webhook, you need to ac
 
 Follow up the acknowledgement with a delta query for the latest changes, and your app should be up to date.  If you are expecting heavy traffic patterns on a particular drive, you should consider calling delta query at a reduced interval rather than after each change notification.  Also, make sure to store the new value returned in the deltaLink parameter to get a new token for calling the API again.
 
+If your processing requires downloading the contents of an individual file, you can use the cTag property to determine if the contents of the file have changed since the last time you downloaded it.  Once you know you want to download the file, you can access the [/content][] property of the DriveItem returned in a delta query response.
+
+### Scanning permissions hierarchies
+
+By default, the delta query response will include sharing information for all items in the query that changed even if they inherit their permissions from their parent and did not have direct sharing changes themselves.  This typically then results in a follow up call to get the permission details for every item rather than just the ones whose sharing information changed.  You can optimize your understanding of how permission changes happen by adding the "Prefer: hierarchicalsharing" header to your delta query request.
+
+When the "Prefer hierarchicalsharing" header is provided, sharing information will be returned for the root of the permissions hierarchy, as well as items that explicitly have sharing changes.  In cases where the sharing change is to remove sharing from an item you will find an empty sharing facet to differentiate between items that inhereit from their parent and those that are unique but have no sharing links.  You will also see this empty sharing facet on the root of a permission hierarchy that is not shared to establish the initial scope.  
```

Comment:
This should be `"Prefer: hierarchicalsharing"` to be consistent with the statement above.

Reply:
"inherit" rather than "inhereit"